modules/alloy/src/main/ts/ephox/alloy/behaviour/positioning/PositionApis.ts

```diff
@@ -29,21 +29,28 @@ const getRelativeOrigin = (component: AlloyComponent): Origins.OriginAdt => {
   return Origins.relative(position.left, position.top, bounds.width, bounds.height);
 };
 
-const place = (component: AlloyComponent, origin: Origins.OriginAdt, anchoring: Anchoring, getBounds: Optional<() => Bounds>, placee: AlloyComponent): void => {
+const place = (
```

Comment:
This is an API breaking change, so make sure you include a major version bump in `versions.txt` for alloy as well please and a changelog entry for alloy.

- [ ] versions.txt entry
- [ ] changelog entry